util/chplenv/chpl_llvm.py

```diff
@@ -14,7 +14,7 @@ def llvm_versions():
     # Which major release - only need one number for that with current
     # llvm (since LLVM 4.0).
     # These will be tried in order.
-    return ('12', '11',)
+    return ('13', '12', '11',)
```

Comment:
is 13 good to go? Or do we need to guard it behind an option?

Reply:
I added it while testing this and forgot to remove it again. Removed now. Thanks.